assets/src/scripts/components/hydrograph/parameters.js

```diff
@@ -129,12 +131,21 @@ export const plotSeriesSelectTable = function (elem, {availableTimeseries, layou
                 // under the appropriate column headers
                 if (!screenSizeCheck) {
                     tr.append('td')
-                        .html(parm => parm[1].currentYear ? '<i class="fa fa-check" aria-label="Current year data available"></i>' : '');
+                        .html(parm => {
+                            const subScript = parm[1].currentYear > 1 ? `<sub>${parm[1].currentYear}</sub>` : '';
```

Comment:
Might be more concise if we have a local function which creates the html. We also lose the subscripts in the tooltip

Reply:
That's not what I meant. We could have a local function that creates the checkmark with the appropriate subscript.

Reply:
I caught that - but I was wondering what you meant by the reference to tooltip

Reply:
Must have missed it. The subscripts are on the small screen tooltip.

Reply:
Ah, I see. Yep, one more thing..